integration/single-node-slug-deploy/check.go

```diff
@@ -348,3 +365,34 @@ func verifyHelloRunning() error {
 		return nil
 	}
 }
+
+func verifyHealthChecks(config *preparer.PreparerConfig, services []string) error {
+	// do consulGet: key = hello
+	opts := kp.Options{
+		Address: config.ConsulAddress,
+		HTTPS:   false,
+	}
+	store := kp.NewConsulStore(opts)
+
+	time.Sleep(5 * time.Second)
+	// check consul for health information for each app
+	for _, sv := range services {
+		name, err := os.Hostname()
+		if err != nil {
+			return err
+		}
+		res, err := store.GetHealth(sv, name)
+		if err != nil {
+			return err
+		} else if res == "" {
+			err = fmt.Errorf("No results for %s", sv)
+			return err
+		} else {
+			fmt.Println(res)
```

Comment:
Next PR: change this to actually verify that services are _healthy_, not just _present_. Needs client integration.